apps/docs/src/content/docs/getting-started/initial-setup.mdx

```diff
@@ -13,6 +13,61 @@ import importedTwUniversalCondig from '@/code-samples/tw-universal-config.js?raw
 import importedTwNativeCondig from '@/code-samples/tw-native-config.js?raw';
 
 <Tabs>
+<TabItem label="CLI">
+
+## Overview
+
+You can quickly set up your project using the React Native Reusables CLI. This will install all necessary dependencies and configure your project automatically.
```

Comment:
Can you add that it is for an existing project:

`You can quickly set up your existing project using the React Native Reusables CLI. This command will install all necessary dependencies and automatically configure your project for you.
`